prometheus.yaml

```diff
@@ -0,0 +1,10 @@
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+
+scrape_configs:
+  - job_name: confio_relayer
+
+    scrape_interval: 5s
+    static_configs:
+      - targets: ['host.docker.internal:26660']
```

Comment:
This is an example configuration file. Is this the right place to store it?

Reply:
Good to commit it to this repo, so it is maintained.

You could put it in a `samples` or `ops` dir. We can add example `systemd` and other configs there as needed.
(Maybe we move it together with the `registry.conf` file?)

Reply:
I realized that `init` command has `registry.yaml` hardcoded to main branch, so I cannot move registry under `samples`. https://github.com/confio/ts-relayer/blob/d8a594bcb602a32a0433608bff90f5629322bb6d/src/binary/ibc-setup/commands/init.ts#L49

One way to "fix" this would be to use parametrized link with version:
```ts
`https://raw.githubusercontent.com/confio/ts-relayer/tree/v${packageJson.version}/demo/registry.yaml`
```

But it fixes the future versions only. Everything till `0.1.3` included will break.

👉  For now, I moved `prometheus.yaml` under `demo`. 92287b8

Reply:
I think the update to fixed version is good.
And let's keep it in demo for now (marked deprecated), and make the main copy in another directory (final location)